x-pack/plugins/apm/server/routes/service_groups/get_service_group_alerts.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kqlQuery } from '@kbn/observability-plugin/server';
+import {
+  AggregationsCardinalityAggregate,
+  AggregationsFilterAggregate,
+  QueryDslQueryContainer,
+} from '@elastic/elasticsearch/lib/api/types';
+import { Logger } from '@kbn/core/server';
+import { ApmPluginRequestHandlerContext } from '../typings';
+import { SavedServiceGroup } from '../../../common/service_groups';
+import { ApmAlertsClient } from './get_apm_alerts_client';
+
+export async function getServiceGroupAlerts({
+  serviceGroups,
+  apmAlertsClient,
+  context,
+  logger,
+  spaceId,
+}: {
+  serviceGroups: SavedServiceGroup[];
+  apmAlertsClient: ApmAlertsClient;
+  context: ApmPluginRequestHandlerContext;
+  logger: Logger;
+  spaceId?: string;
+}) {
+  if (!spaceId || serviceGroups.length === 0) {
+    return {};
+  }
+  const serviceGroupsKueryMap: Record<string, QueryDslQueryContainer> =
+    serviceGroups.reduce((acc, sg) => {
+      return {
+        ...acc,
+        [sg.id]: kqlQuery(sg.kuery)[0],
+      };
+    }, {});
+  const params = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          { term: { 'kibana.alert.rule.producer': 'apm' } },
+          { term: { 'kibana.alert.status': 'active' } },
```

Comment:
WDYT about storing the field names as constants, similar to `elasticsearch_fieldnames.ts`?